spec.md

```diff
@@ -471,6 +471,7 @@ how Platforms might expose these values to their users.
 | metadata | object | An opaque object of metadata for a Service Plan. It is expected that Platforms will treat this as a blob. Note that there are [conventions](profile.md#service-metadata) in existing Service Brokers and Platforms for fields that aid in the display of catalog data. |
 | free | boolean | When false, Service Instances of this plan have a cost. The default is true. |
 | bindable | boolean | Specifies whether Service Instances of the Service Plan can be bound to applications. This field is OPTIONAL. If specified, this takes precedence over the `bindable` attribute of the service. If not specified, the default is derived from the service. |
+| updateable | boolean | Whether the Plan supports upgrade/downgrade/sidegrade to another version. Please note that the attribute is intentionally misspelled as `updateable` for legacy reasons. If not specificed the Service Offering's `updateable` field applies. |
```

Comment:
We should add precdence behaviour if updateable is specified at the service level too. We did this for bindable and added the text:

`If specified, this takes precedence over the `bindable` attribute of the service. If not specified, the default is derived from the service.`